rsa_pss_test.go

```diff
@@ -1,10 +1,11 @@
+//go:build go1.4
```

Comment:
Do we still need these build tags? Can we remove them entirely..

Reply:
I think that's out of scope for this PR, but I'm happy to create another PR removing them. This build line is added because [Go 1.17](https://go.dev/doc/go1.17) has a new syntax for them.

Reply:
That's fair.